spec/alephant/broker/cache/cached_object_spec.rb

```diff
@@ -0,0 +1,152 @@
+require "spec_helper"
+
+RSpec.describe Alephant::Broker::Cache::CachedObject do
+  subject { described_class.new(s3_obj) }
+
+  let(:last_modified) { Time.parse("Mon, 11 Apr 2016 10:39:57 GMT") }
+  let(:ttl)           { 15 }
+
+  let(:s3_obj) do
+    AWS::Core::Data.new(
```

Comment:
Interesting, you've created a new instance of an internal AWS class (well, maybe not _internal_ but effectively something lower level than we typically worry about).

I think in this situation I would've passed in an `instance_double`, but nothing wrong with this per se. Although I feel like an instance_double would be more idiomatic.